{
  "commands": [
    {
      "commandType": "home",
      "notes": [],
      "params": {},
      "result": {},
      "status": "succeeded"
    }
  ],
  "config": {
    "apiVersion": [
      2,
<<<<<<< HEAD
      16
=======
      18
>>>>>>> 929e01421e (fix(test): snapshot renaming (#15038))
    ],
    "protocolType": "python"
  },
  "errors": [
    {
<<<<<<< HEAD
      "detail": "ValueError [line 15]: Staging areas not permitted for trash bin.",
=======
      "detail": "TypeError [line 11]: ParameterContext.add_str() got an unexpected keyword argument 'unit'",
>>>>>>> 929e01421e (fix(test): snapshot renaming (#15038))
      "errorCode": "4000",
      "errorInfo": {},
      "errorType": "ExceptionInProtocolError",
      "wrappedErrors": [
        {
<<<<<<< HEAD
          "detail": "ValueError: Staging areas not permitted for trash bin.",
          "errorCode": "4000",
          "errorInfo": {
            "args": "('Staging areas not permitted for trash bin.',)",
            "class": "ValueError",
<<<<<<< HEAD:app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[58750bf5fb][Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4].json
<<<<<<< HEAD:app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[58750bf5fb][Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4].json
            "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line 124, in run_python\n    exec(\"run(__context)\", new_globs)\n\n  File \"<string>\", line 1, in <module>\n\n  File \"Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4.py\", line 15, in run\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/api_support/util.py\", line 383, in _check_version_wrapper\n    return decorated_obj(*args, **kwargs)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/protocol_context.py\", line 512, in load_trash_bin\n    raise ValueError(\"Staging areas not permitted for trash bin.\")\n"
=======
            "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line 112, in run_python\n    exec(\"run(__context)\", new_globs)\n\n  File \"<string>\", line 1, in <module>\n\n  File \"Flex_None_None_2_16_AnalysisError_TrashBinInStagingAreaCol4.py\", line 15, in run\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/api_support/util.py\", line 383, in _check_version_wrapper\n    return decorated_obj(*args, **kwargs)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/protocol_context.py\", line 508, in load_trash_bin\n    raise ValueError(\"Staging areas not permitted for trash bin.\")\n"
>>>>>>> ca2f7d91f0 (fix(app-testing): snapshot failure capture (#14586)):app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[ac35bb394d][Flex_None_None_2_16_AnalysisError_TrashBinInStagingAreaCol4].json
=======
            "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line 124, in run_python\n    exec(\"run(__context)\", new_globs)\n\n  File \"<string>\", line 1, in <module>\n\n  File \"Flex_None_None_2_16_AnalysisError_TrashBinInStagingAreaCol4.py\", line 15, in run\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/api_support/util.py\", line 383, in _check_version_wrapper\n    return decorated_obj(*args, **kwargs)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/protocol_context.py\", line 508, in load_trash_bin\n    raise ValueError(\"Staging areas not permitted for trash bin.\")\n"
>>>>>>> f4443ad9d0 (fix(app-testing): snapshot failure capture (#14756)):app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[ac35bb394d][Flex_None_None_2_16_AnalysisError_TrashBinInStagingAreaCol4].json
=======
          "detail": "TypeError: ParameterContext.add_str() got an unexpected keyword argument 'unit'",
          "errorCode": "4000",
          "errorInfo": {
<<<<<<< HEAD:app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[89a255db0b][OT2_X_v2_18_None_None_StrRTPwith_unit].json
            "args": "(\"ParameterContext.add_str() got an unexpected keyword argument 'unit'\",)",
            "class": "TypeError",
            "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line 80, in _parse_and_set_parameters\n    exec(\"add_parameters(__param_context)\", new_globs)\n\n  File \"<string>\", line 1, in <module>\n\n  File \"OT2_X_v2_18_None_None_StrRTPwith_unit.py\", line 11, in add_parameters\n"
=======
            "args": "('Staging areas not permitted for trash bin.',)",
            "class": "ValueError",
<<<<<<<< HEAD:app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[89a255db0b][OT2_X_v2_18_None_None_StrRTPwith_unit].json
            "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line 124, in run_python\n    exec(\"run(__context)\", new_globs)\n\n  File \"<string>\", line 1, in <module>\n\n  File \"Flex_None_None_2_16_AnalysisError_TrashBinInStagingAreaCol4.py\", line 15, in run\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/api_support/util.py\", line 383, in _check_version_wrapper\n    return decorated_obj(*args, **kwargs)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/protocol_context.py\", line 512, in load_trash_bin\n    raise ValueError(\"Staging areas not permitted for trash bin.\")\n"
>>>>>>> c3ee8d545c (fix(app-testing): snapshot failure capture (#14813)):app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[ac35bb394d][Flex_None_None_2_16_AnalysisError_TrashBinInStagingAreaCol4].json
========
            "traceback": "  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/execution/execute_python.py\", line 124, in run_python\n    exec(\"run(__context)\", new_globs)\n\n  File \"<string>\", line 1, in <module>\n\n  File \"Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4.py\", line 15, in run\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocols/api_support/util.py\", line 383, in _check_version_wrapper\n    return decorated_obj(*args, **kwargs)\n\n  File \"/usr/local/lib/python3.10/site-packages/opentrons/protocol_api/protocol_context.py\", line 512, in load_trash_bin\n    raise ValueError(\"Staging areas not permitted for trash bin.\")\n"
>>>>>>>> 929e01421e (fix(test): snapshot renaming (#15038)):app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[58750bf5fb][Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4].json
>>>>>>> 929e01421e (fix(test): snapshot renaming (#15038))
          },
          "errorType": "PythonException",
          "wrappedErrors": []
        }
      ]
    }
  ],
  "files": [
    {
<<<<<<< HEAD
      "name": "Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4.py",
=======
<<<<<<<< HEAD:app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[89a255db0b][OT2_X_v2_18_None_None_StrRTPwith_unit].json
      "name": "OT2_X_v2_18_None_None_StrRTPwith_unit.py",
========
      "name": "Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4.py",
>>>>>>>> 929e01421e (fix(test): snapshot renaming (#15038)):app-testing/tests/__snapshots__/analyses_snapshot_test/test_analysis_snapshot[58750bf5fb][Flex_X_v2_16_NO_PIPETTES_TrashBinInStagingAreaCol4].json
>>>>>>> 929e01421e (fix(test): snapshot renaming (#15038))
      "role": "main"
    },
    {
      "name": "cpx_4_tuberack_100ul.json",
      "role": "labware"
    },
    {
      "name": "opentrons_ot3_96_tiprack_1000ul_rss.json",
      "role": "labware"
    },
    {
      "name": "opentrons_ot3_96_tiprack_200ul_rss.json",
      "role": "labware"
    },
    {
      "name": "opentrons_ot3_96_tiprack_50ul_rss.json",
      "role": "labware"
    },
    {
      "name": "sample_labware.json",
      "role": "labware"
    }
  ],
  "labware": [],
  "liquids": [],
  "metadata": {
<<<<<<< HEAD
    "author": "Derek Maggio <derek.maggio@opentrons.com>",
    "protocolName": "QA Protocol - Analysis Error - Trash Bin in Staging Area Column 4"
  },
  "modules": [],
  "pipettes": [],
  "robotType": "OT-3 Standard",
=======
    "protocolName": "Str RTP with unit"
  },
  "modules": [],
  "pipettes": [],
  "robotType": "OT-2 Standard",
>>>>>>> 929e01421e (fix(test): snapshot renaming (#15038))
  "runTimeParameters": []
}
